website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -71,7 +71,7 @@ A draft schema for how the objects would interact and be tied together can be
 seen below.
 
 <MyImage
-    imageSource={useBaseUrl('img/Research-Reconciliation-orca-inventory-schema.png')}
+    imageSource={useBaseUrl('img/Research-Reconciliation-orca-inventory-schema-updated.png')}
```

Comment:
not sure why renaming here. Just overwrite the original file. I would convert back and rename your file and over write the original.